testbench/grpc_server.py

```diff
@@ -463,6 +464,7 @@ def QueryWriteStatus(self, request, context):
         )
 
     def GetServiceAccount(self, request, context):
+        request.project[len("projects/")]
```

Comment:
Not sure what this line does... same on L487

Reply:
Nothing useful, of course. Removed, and thanks.